addons/website_blog/views/website_blog_posts_loop.xml

```diff
@@ -39,8 +39,8 @@ according to the enabled options.
             <!-- Filters -->
             <div t-if="tag or date_begin or search" class="col-12 mb-3">
                 <div t-if="posts" class="h4 mb-3">
-                    <t t-esc="len(posts)"/>
-                    <t t-if="len(posts) &lt; 2">Article</t>
+                    <t t-esc="search_count"/>
```

Comment:
This is wrong.
You can quickly have a functional check with the tags, which works fine to see how it (is supposed to) works.
- Duplicate 15 times the "Don't look up" blog post
- Click on the discovery tag, you land on something like `/blog/tag/discovery-5`

You see 12 articles, and on page 2 4 articles. **You don't see 16 articles.**

You are doing a functional change for no reason in stable, which is terrible as it will suddenly change existing blogs.

Arguably, you can think it is better to show the total amount instead of the page amount (I tend to agree with that), but that's not a fix at all, it would be an improvement. Nothing is wrong here?

Reply:
It was requested from the ticket.
But, I didn't see it from your POV. And I do agree with what you say. 
Let me delete that out.

Reply:
So for this kind of [IMP] should I create another ticket or just let it be that way?

Reply:
Yep, I'll contact them.
Thanks for the info!